airbyte-webapp/src/config/frequencyConfig.ts

```diff
@@ -0,0 +1,35 @@
+import { ConnectionSchedule } from "core/request/AirbyteClient";
+
+const frequencyConfig: Array<ConnectionSchedule | null> = [
```

Comment:
We usually try to have named exports (the `default` exports) are left overs we're trying to eliminate. So since we're anyway touching this, could we please turn this into a:

```
export const frequencyConfig ...
```

instead of the default export at the bottom.

Reply:
Ah yeah, this was mentioned in the style guide too - my bad! I've pushed this change